src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs

```diff
@@ -180,6 +180,12 @@ public IService ParentService
             }
         }
 
+        // Used in tests
+        internal (string Namespace, string Name) GetName()
+        {
+            return (base.Namespace.Name.Value, base.Name);
```

Comment:
If it's just in those tests, can't you change the tests to, for example: 

 

func =>
                {
                    return
                        func.Namespace.Name.Value == "LQA" &&
                        func.Name == "GetAnswersFromText";
                }

Reply:
Updated it to add internals visibility to the connectors test project